packages/help-center/src/components/back-button.tsx

```diff
@@ -24,8 +26,13 @@ export const BackButton = ( { onClick, backToRoot = false, className }: Props )
 			navigate( -1 );
 		}
 	}
+
+	useEffect( () => {
+		buttonRef.current?.focus?.();
+	}, [] );
+
 	return (
-		<Button className={ buttonClassName } onClick={ onClick || defaultOnClick }>
+		<Button className={ buttonClassName } ref={ buttonRef } onClick={ onClick || defaultOnClick }>
```

Comment:
Isn't this the equivalent of 

```suggestion
		<Button className={ buttonClassName } autoFocus onClick={ onClick || defaultOnClick }>
```

I know the linter will complain, but I would add a linting ignore instead of managing refs.

Reply:
I think `autoFocus` can be inconsistent based on render timings? It may be be fine in this situation though.

Reply:
Yes exactly. But since the effect has `[]` deps, I think it would have the same effect here.

Reply:
Sounds good 👍